### PR TITLE
add options to customize the model view dashboard

### PR DESCRIPTION
**Pull request by alanrenmsft**
regardless whether we are going to use this component, it is good expose these options.

**showTabsWhenOne: true, showIcon: true**
Screen Shot 2020-04-06 at 3 04 29 PM

**showTabsWhenOne: true, showIcon: false**
Screen Shot 2020-04-06 at 3 03 33 PM

**showTabsWhenOne: false** - Default behavior
Screen Shot 2020-04-06 at 3 05 11 PM

### CODE REVIEW

File: src/sql/azdata.proposed.d.ts [hunk 1]
```diff
@@ -321,5 +322,17 @@ declare module 'azdata' {
...
+	export interface ModelViewDashboardOptions {
+		/**
+		 * Whether to show the tab icon, default is true
+		 */
+		showIcon?: boolean;
+
+		/**
+		 * Whether to show the tab header when there is only one tab, default is false
+		 */
+		showTabsWhenOne?: boolean;
```
Comment (chlafreniere) [inline src/sql/azdata.proposed.d.ts:335]: hmm this name is a little awkward. can you work with PM?

Comment (Charles-Gagnon) [reply src/sql/azdata.proposed.d.ts:335]: What if you just made it "alwaysShowNavigationPane" (default false) - then users would just set that to true if they wanted to show it even when 1 tab is shown

Comment (Charles-Gagnon) [reply src/sql/azdata.proposed.d.ts:335]: And also - I don't really consider the tabs in the navigation pane to be "headers" so I'd consider rewording that.

File: src/sql/workbench/api/common/extHostModelViewDialog.ts [hunk 2]
```diff
@@ -481,7 +482,8 @@ class ModelViewDashboardImpl implements azdata.window.ModelViewDashboard {
 
 			const tabbedPanel = view.modelBuilder.tabbedPanel().withTabs(tabs).withLayout({
 				orientation: 'vertical',
-				showIcon: true
+				showIcon: (this._options && this._options.showIcon) ?? true,
```
Comment (Charles-Gagnon) [inline src/sql/workbench/api/common/extHostModelViewDialog.ts:485]: You can shorten this to just `this._options?.showIcon ?? true`

File: src/sql/azdata.proposed.d.ts (hunk 1 above)
Comment (alanrenmsft) [reply src/sql/azdata.proposed.d.ts:335]: i am renaming it to alwaysShowTabs.

**Comment by alanrenmsft** [comment]
my favorite PM @yualan said the new name looks good to him. 😆
